openslides_backend/action/action.py

```diff
@@ -193,9 +191,8 @@ def check_permissions(self, instance: Dict[str, Any]) -> None:
                 ):
                     return
                 raise MissingPermission(self.permission)
-        else:
-            if self.permission_service.is_allowed(self.name, self.user_id, [instance]):
-                return
+        elif is_dev_mode():
```

Comment:
Please remove this, ignoring permission errors in dev mode might lead to unnoticed errors.